tools/update-gcp-infra.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# Turbinia GCP management script
+# This script can be used to manage a Turbinia stack deployed to GCP. The only requirement is you have 'gcloud' installed.
+# Use --help to show you commands supported.
+
+set -o posix
+
+CMD=$1 
+INSTANCEID=$2
+ZONE=$3
+DOCKER_TAG=$4
+CONFIG_FILE=$4
+
+SERVER_URI="us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server"
+WORKER_URI="us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-worker"
+GCLOUD=`command -v gcloud`
+
+function usage { 
+  echo "Usage: $0 COMMAND INSTANCEID ZONE [DOCKER_TAG or CONFIG_FILE]" 
+  echo "COMMAND         Pick one of the commands below"
+  echo "INSTANCEID      The Turbinia deployment instance ID"
```

Comment:
We could also assume that there is a .turbiniarc file somewhere in the standard places and use `tools/turbinia_config_parser.py` to load the right config file and extract the INSTANCE id.

Reply:
Could do the same for project id and zone too, that way we don't have to rely on the gcloud defaults.

Reply:
So the reason I did not do this is because the configuration file is big and contains lots of other options you don't need. With 2 cli parameters (zone and instance id) you can use this script without the need to have any Turbinia configuration file/knowledge. I would prefer to keep this simple and only rely on the gcloud binary.